<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>‚ÑπÔ∏è Sobre o Planejador Financeiro Pessoal/ About the Personal Finance Planner</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>
       &lt;h2&gt;üíµ Personal Finance Planner&lt;/h2&gt;
       &lt;p&gt;&lt;b&gt;Carlos Eduardo Mulinari&lt;/b&gt;&lt;br&gt;UPF ‚Äî Engenharia da Computa√ß√£o ‚Äî 2025&lt;/p&gt;
       &lt;p&gt;Planejador financeiro inteligente com Arduino e Qt/ Intelligent Finance Planner with Arduino and Qt.&lt;/p&gt;
      </string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="githubButton_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Abrir reposit√≥rio no GitHub</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: transparent;
             border: none;
         }
         QPushButton:hover {
             transform: scale(1.1);
         }
        </string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/github.png</normaloff>:/github.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="youtubeButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Assistir ao v√≠deo no YouTube</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: transparent;
             border: none;
         }
         QPushButton:hover {
             transform: scale(1.1);
         }
        </string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/youtube.png</normaloff>:/youtube.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QPushButton {
           background-color: #e53935;
           color: white;
           font: bold 12pt &quot;Segoe UI&quot;;
           border-radius: 8px;
           padding: 6px;
       }
       QPushButton:hover {
           background-color: #c62828;
       }
      </string>
     </property>
     <property name="text">
      <string>‚ùå Fechar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
